runtests.py

```diff
@@ -168,6 +168,50 @@ def store_variables(assignments, source, ns):
         else:
             variables.pop(variable, None)
 
+    
+
+def apply_xslt(row_number, row):
+    """
+    Process a CSV row as an XSLT method.
+    Payload is transformed by URI. Transformed files are stored in paths (described 
+        as /path/to/file=*) within Store. Rest of Store assignments are recorded.
+    row_number: CSV row number
+    row: Dictionary object that contains csv row data
+    """
+    xml_file, xslt_file = row['Payload'], row['URI']
+    
+    # separate outputs and assignments
+    output_files = []
+    assignments = []
+    for line in row['Store'].splitlines():
+        lhs, rhs = line.split('=', 1)
+        if rhs == '*':
+            output_files.append(lhs)
+        else:
+            assignments.append(line)
+    
+    assignments = '\n'.join(assignments)
+
+    with open(xml_file, 'rb') as source_file:
+        xml_source = source_file.read()
+    with open(xslt_file, 'rb') as source_file:
+        xslt_source = source_file.read()
+       
+    namespaces = string_to_dictionary(row['NS'])
+    store_variables(assignments, xml_source, namespaces)
+    store_variables(assignments, xslt_source, namespaces)
+    
+    xml = etree.fromstring(xml_source)
+    xsl = etree.fromstring(xslt_source)
+    transform = etree.XSLT(xsl)
+    output = transform(xml)
+
+    for output_file in output_files:
+        with open(output_file, 'w') as f:
+            output.write(output_file)
+
```

Comment:
The QA file also included some Test operations which are not covered here.  If we extract the Test operations from `test_http_request` and call them in both HTTP and XSLT operations, everything has Test superpowers.